bot/exts/utils/snekio.py

```diff
@@ -0,0 +1,64 @@
+"""I/O File protocols for snekbox."""
```

Comment:
The convention for handling extension-specific utilities is to convert the extension into a subpackage and put related modules in there. See `help_channels`, `doc`, and `codeblock` for examples. Every non-extension module's name has to start with `_` so the extension loader knows to skip it.

Since this module is specific to snekbox, the above should be applied to the snekbox extension. I also suggest renaming this module to just `io` once it's moved, since it'll already be namespaced in the `snekbox` module. Or will that conflict with stdlib `io` still?

Reply:
Moved to `utils/snekbox/_io.py` in [727a146](https://github.com/python-discord/bot/pull/2326/commits/727a146f2de0f37c43d6939dc4368ef780373cd4)